.github/workflows/probe.yaml

```
@@ -0,0 +1,38 @@
+name: Build and push probe image.
```

nitpick: I have noticed that other workflow names are without a dot at the end.
```suggestion
name: Build and push probe image
```